src/form-input-message-group.scss

```diff
@@ -9,6 +9,5 @@
 $block: #{$fd-namespace}-form-input-message-group;
 
 .#{$block} {
-  flex: 1;
```

Comment:
I think you forgot to add `display: flex`.
You additionally need to specify `flex-grow: 1`, to stretch inline labels and inputs.

![image](https://user-images.githubusercontent.com/17496353/71174531-b7dd5b00-2265-11ea-9279-b06967d64439.png)